Kitodo/src/test/java/org/kitodo/production/forms/ProcessFormIT.java

```diff
@@ -109,4 +113,30 @@ public void testProcessesSelection() throws Exception {
                 .stream().map(Process::getId).sorted().collect(Collectors.toList());
         assertEquals(new ArrayList<>(Arrays.asList(1, 2, 5)), selectedIds);
     }
+
+    @Test
+    public void testSortHelperStatusIsUpdatedOnTaskSave() throws Exception {
```

Comment:
Personally I would use here a `DAOException` instead of the general `Exception` class as only `DAOException`s could be thrown and if other exceptions are thrown they may let the test be successful too.